### PR TITLE
[ASM] - EXPANDER - 4378 - NotificationTicketType Input for ServiceNow

### PR DESCRIPTION
**Pull request by BigEasyJ**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues

[EXPANDR-4378](https://jira-dc.paloaltonetworks.com/browse/EXPANDR-4378)

## Description

Updated playbooks to use new playbook input, `NotificationTicketType`, for ServiceNow ticket type.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by capanw — APPROVED** [review]

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml [hunk 1]
```diff
@@ -3159,6 +3162,12 @@ inputs:
   required: true
   description: Body of the notification (email or ticket) sent to potential service owner.
   playbookInputQuery:
+- key: NotificationTicketType
+  value:
+    simple: incident
+  required: true
+  description: The ticket type used by ticketing systems. The default is "incident" because ticketing systems such as ServiceNow and Cherwell use it as default.
```
[the remark this answers is not in the document]
Comment (BigEasyJ) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml:3169]: Is there such a resource? For Jira and ServiceNow they are both customizable so there is not a published list for either that I have seen.

- [Jira reference](https://community.atlassian.com/t5/Jira-questions/Is-there-a-complete-list-of-issue-types-There-are-many-more-than/qaq-p/2053879)
- [SNow reference](https://www.servicenow.com/community/developer-forum/where-we-can-find-list-of-tables-in-servicenow/m-p/1587266)

[the remark this answers is not in the document]
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml:3169]: apologies, I thought there were defined lists

**Review by johnnywilkes — APPROVED** [review]
@BigEasyJ , good stuff!

**Comment by ilappe** [comment]
Hi @johnnywilkes 
are this ready for XSOAR review?

**Comment by ilappe** [comment]
Hi @BigEasyJ 
what's the status here?

**Comment by ilappe** [comment]
@BigEasyJ @johnnywilkes 
a gentleman reminder...

**Comment by ilappe** [comment]
as discussed in Slack
this are ready to merge
i will fix the validation issue in the internal PR

**Review by ilappe — APPROVED** [review]

**Comment by BigEasyJ** [comment]
Conflicts have been fixed and merged 👍🏽

**Pull request merged**